x-pack/plugins/security_solution/public/detections/components/rules/data_view_selector/translations.tsx

```diff
@@ -30,3 +30,18 @@ export const DATA_VIEW_NOT_FOUND_WARNING_DESCRIPTION = (dataView: string) =>
         'Your data view of "id": "{dataView}" was not found. It could be that it has since been deleted.',
     }
   );
+
+export const DDATA_VIEW_ALERTS_ON_ALERTS_WARNING_LABEL = i18n.translate(
+  'xpack.securitySolution.detectionEngine.stepDefineRule.dataViewIncludesAlertsIndexLabel',
+  {
+    defaultMessage: 'Default security data view',
+  }
+);
+
+export const DATA_VIEW_ALERTS_ON_ALERTS_WARNING_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.detectionEngine.stepDefineRule.dataViewIncludesAlertsIndexDescription',
+  {
+    defaultMessage:
+      'The default security data view includes the alerts index. This could result in getting alerts off of existing alerts.',
```

Comment:
Would "duplicate" be accurate here? Or another descriptor, like "additional" or "redundant"?
```suggestion
      'The default Security data view includes the alerts index. This could result in duplicate alerts being generated from existing alerts.',
```

Reply:
I think "redundant" could work here.

Reply:
Updated text and pushed up. Let me know what you think!